src/config.ts

```diff
@@ -197,11 +197,17 @@ export class KubeConfig {
             scheme = 'http';
         }
 
+        // Wrap raw IPv6 addresses in brackets.
+        let serverHost = host;
+        if (host && host.indexOf(':') != -1) {
```

Comment:
There are methods to check if its an ipv6 Adress.

Reply:
done.